README.md

```diff
@@ -3,9 +3,10 @@
 
 # WeakAuras 2
 
-WeakAuras is a powerful and flexible framework that allows you to display highly customizable graphics on your screen to indicate buffs, debuffs, and a whole host of similar types of information. It was originally meant to be a lightweight replacement for Power Auras, but it now incorporates many features that Power Auras does not, while still remaining more efficient and easy to use.
+WeakAuras is a powerful and flexible framework that allows the display of highly customizable graphics on WoW's user interface to indicate buffs, debuffs, and other relevant information. This addon was created to be a lightweight replacement for Power Auras but has since introduced more functionalities while remaining efficient and easy to use.
```

Comment:
“the WoW UI”, or perhaps “the World of Warcraft user interface” (if you prefer not to abbreviate UI) would be preferable instead of “WoW’s user interface”

Reply:
“This addon was created to be ...” -> “WeakAuras began as...”

Reply:
Comma required between “Power Auras” and “but” in the final sentence

Reply:
Also “easy to use” is debatable, but perhaps that is beyond the scope of this PR...

Reply:
Thanks for your suggestions!

As I was taught in class there is no definitive way of writing so I am always learning!
I understand technical writing sounds picky and should not be applied elsewhere other than formal documentation.
Please let me go through my thinking process about the issues you pointed out.

1. I used 'WoW'  because the scope of the addon is contained within World of Warcraft. So I assume readers will understand this abbreviation. However, UI is a more technical term so I used 'user interface'. You are right, using the full form increases formality.

2. I used the term the 'addon was created' because WeakAura didn't technically 'began'. You guys, the creators, began working on the project to replace PowerAura. 

3. The comma is required if 'but' links independent clauses. The phrase 'but has since introduced more functionalities...' is an incomplete sentence so the comma for the compound predicate is okay to be left out.

4. I know WeakAuras can be difficult to configure at times. 'Easy to use' is in the original document and I don't want to create any bias by removing it.

Thanks for hearing me out!